src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -406,7 +408,7 @@ void createConsumerWithForwardedHeaderDefaultPort(VertxTestContext context) thro
     }
 
     @Test
-    void createConsumerWithForwardedHeaderWrongProto(VertxTestContext context) throws InterruptedException {
+    void createConsumerWithForwardedHeaderWrongProto(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
```

Comment:
which part of the code raises these new two exceptions?

Reply:
old code sorry :(